xpdan/tests/test_tiff_exporter.py

```diff
@@ -0,0 +1,26 @@
+import os
+import tempfile
+from tifffile import imread
+from xpdan.tests.conftest import img_size
+from xpdan.callbacks_core import XpdAcqLiveTiffExporter
+
+# standard config
+base = tempfile.mkdtemp()
+template = os.path.join(base, 'xpdUser/tiff_base/')
```

Comment:
Can you use a pytest fixture for this? That way after the test is finished the folder can be removed.